proc/syscall_windows.go

```diff
@@ -22,8 +22,53 @@ type _THREAD_BASIC_INFORMATION struct {
 	BasePriority   int32
 }
 
+type _CREATE_PROCESS_DEBUG_INFO struct {
+	File                syscall.Handle
+	Process             syscall.Handle
+	Thread              syscall.Handle
+	BaseOfImage         uintptr
+	DebugInfoFileOffset uint16
+	DebugInfoSize       uint16
+	ThreadLocalBase     uintptr
+	StartAddress        uintptr
+	ImageName           *byte
```

Comment:
I will take your word for it and change both "ImageName" into uintptr. Thanks for the article - my weekend reading.